### PR TITLE
Updating the IRIs used in RDF

### PR DESCRIPTION
**Pull request by zvr**
This simply updates the documentation on RDF,
to correctly match the IRIs used in the model,
as present in each namespace file.

### CODE REVIEW

File: serialization/rdf.md [hunk 1]
```diff
@@ -2,10 +2,19 @@
 
 SPDX data can be serialized in RDF. This can be saved in a variety of formats, like XML, JSON-LD, Turtle, etc.
 
-1. The namespace for SPDX v3 is `http://spdx.org/rdf/v3`
+1. The namespace for SPDX v3 is `https://rdf.spdx.org/v3`
```
Comment (goneall) [inline serialization/rdf.md:5]: In the current spec (version 2.X) the namespace also is a URL pointing to the documentation.

If we change it to `rdf.spdx.org`, we would need to register the subdomain through the LF.

Is this something we want to do? It make take a bit of work and time to get through the LF IT - but I can make an attempt at it if we want to change it.

**Review by goneall — CHANGES_REQUESTED** [review]

**Comment by zvr** [comment]
The purpose of this PR is simply to harmonize the doc about RDF with the addresses actually used in the model files, so that we have a consistent set that can be presented without raising questions. The alternatives were either change the single doc file or change each and every namespace file. I opted for the easiest path.

We can discuss/decide the actual format at a later stage; I don't expect anyone to start using these for a while.

Regarding what they should be, I agree that they should point to the ontology documentation. There are two implementaion alternatives:
- create a new DNS entry and have it pointing to somewhere where static web content is server; or
- carve out a subtree from the current WordPress setup to serve static content from somewhere.

No predictions on which of the two will be easier with LF... ;-)  We can ask and decide later.

But again, this is only to have everything aligned.

File: serialization/rdf.md (hunk 1 above)
Comment (goneall) [reply serialization/rdf.md:5]: I opened a separate issue to track

**Review by goneall — APPROVED** [review]

**Comment by goneall** [comment]
> The purpose of this PR is simply to harmonize the doc about RDF with the addresses actually used in the model files, so that we have a consistent set that can be presented without raising questions. The alternatives were either change the single doc file or change each and every namespace file. I opted for the easiest path.
> 
> We can discuss/decide the actual format at a later stage; I don't expect anyone to start using these for a while.
> 
> Regarding what they should be, I agree that they should point to the ontology documentation. There are two implementaion alternatives:
> 
> * create a new DNS entry and have it pointing to somewhere where static web content is server; or
> * carve out a subtree from the current WordPress setup to serve static content from somewhere.
> 
> No predictions on which of the two will be easier with LF... ;-) We can ask and decide later.
> 
> But again, this is only to have everything aligned.

I opened issue #596 to track resolving this before release of 3.0.  I'll go ahead and approve this change.

**Review by kestewart — APPROVED** [review]
Approved pending securing of web namespace for rdf.spdx.org

**Comment by kestewart** [comment]
Discussed in tech call,  no issues raised, so going ahead and merging.

**Pull request merged**